data/free worlds start.txt

```diff
@@ -1103,6 +1103,83 @@ mission "FW Flamethrower 2"
+	source Rust
+	to offer
+		has "FW Flamethrower Turret 1: done"
+	on offer
+		conversation
+			`You received a message recently from Barmy Edward, asking you to stop by the Kraz Cybernetics facility to help test a new weapon. Do you want to visit them now?`
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					defer
+			`	The Kraz Cybernetics receptionist points you in a familiar direction, towards the hangar where Barmy Edward does his work. You do not find him there; instead, you are greeted by one of his colleagues: "Good afternoon. Captain <last>, is it? I'm Dr. Chris Cardona. I work with Barmy on the occasional project, including this latest prototype. We had some - uh - technical difficulties with the fuel injection component, and Barmy is on temporary medical leave. Not to worry, he left me with clear instructions regarding your assignment."`
+			`	He seems utterly resigned to this state of affairs.`
+			`	Just as before, you are to engage our test ship in combat in orbit above <planet> and attempt to disable it without causing permanent damage to its hull. In this case, it is an upgraded Argosy, the Doombat II."`
+			choice
+				`	"Technical difficulties?"`
+				`	"Medical leave?"`
+			`	"Yes, the fuel injector came loose and doused the hangar with hyperspace fuel. It didn't ignite, but Edward inhaled enough of the vapours to put him out of action for a few weeks. Thankfully, the lab was not damaged, and production of test units is still on schedule."`
+			choice
+				`	"For science!"`
+					accept
+				`	"For Barmy!"`
+					accept
+				`	"Sorry, I'd rather not end up like Barmy. I don't want to be involved."`
+					decline
+	
+	npc save disable
+		personality staying heroic
+		government "Test Dummy"
+		ship "Argosy (Flamethrower Turret)" "Redhammer"
+	
+	on complete
+		event "flamethrower turret available" 70
+		conversation
+			`You return to Kraz Cybernetics to deliver your report to Dr. Cardona. As you approach the hangar, he steps outside, removes a respirator and pair of safety goggles from his face, and rushes forward to meet you.`
+			`	"Captain <last>, you're back so soon? Come, let us discuss your findings in my office, where the air is - uh - better conditioned. It's humid out here, no?"`
+			`	You follow him to his office, away from the hangar, and away from an unusual smell which has begun to fill the air.`
+			`	"The weapon worked well, I presume?"`
+			choice
+				`	"Indeed."`
+					goto good
+				`	"I was able to neutralize it without much effort."`
+					goto bad
+			label good
+			`	"Glad to hear it."`
+			`	He glances at the figures displayed on his monitor.`
+			`	"Yes, the data we've collected remotely seem to meet our expectations."`
+				goto end
+			label bad
+			`	"I see. Surely that speaks more to the effectiveness of your ship's cooling system than to any deficiency in the prototype."`
+			choice
+				`	"Indeed."`
+				`	"Don't call me Shirley."`
```

Comment:
Where does the name Shirley come from? It doesn't seem to fit with the way the game presents the main character (you).